benchopt/runner.py

```diff
@@ -212,32 +172,26 @@ class _Callback:
     ----------
     objective : instance of BaseObjective
         The objective to minimize.
-    max_iter : int
-        Maximum number of iterations to run for.
-    deadline : float
-        Deadline after which to stop the computations. This will be
-        used to respect the timeout for each solver.
     meta : dict
         Metadata passed to store in Cost results.
         Contains objective and data names, problem dimension, etc.
+    stopping_criterion : StoppingCriterion
+        Object to check if we need to stop a solver.
 
     Attributes
     ----------
     objective : instance of BaseObjective
         The objective to minimize.
-    max_iter : int
-        Maximum number of iterations to run for.
-    deadline : float
-        Deadline after which to stop the computations. This will be
-        used to respect the timeout for each solver.
+    stopping_criterion : StoppingCriterion
+        Object to check if we need to stop a solver.
     meta : dict
         Metadata passed to store in Cost results.
         Contains objective and data names, problem dimension, etc.
     info : dict
         A dictionary with info from the current system.
     curve : list
         The convergence curve stored as a list of dict.
-    status : 'done' | 'diverged' | 'timeout' | 'unfinished'
+    status : 'done' | 'diverged' | 'timeout' | 'max_runs'
```

Comment:
`get_results` docstring for `status` has an `unfinished` that should be replaced now.

Reply:
good catch!